providers/scaleway/v1/config.go

```diff
@@ -0,0 +1,91 @@
+package v1
```

Comment:
Would actually keep things simple and place this directly in `providers/scaleway`

Reply:
I can avoid versioning the package, but it relies on API and API are versioned, but yeah we can handle it later in another way